android/java/org/chromium/chrome/browser/settings/BraveSyncScreensPreference.java

```diff
@@ -756,7 +756,7 @@ private void seedWordsReceivedImpl(String seedWords) {
         }
     }
 
-    private void showMainSyncScrypt() {
+    private void showMainSyncScript() {
```

Comment:
I'm not at all certain about this. I understand that there's a thing called [Scrypt](https://en.wikipedia.org/wiki/Scrypt), but there's no indication that this is that.